src/js/services/bootstrap.js

```diff
@@ -77,6 +77,8 @@ export default App.extend({
     return this.workspaces.clone();
   },
   getSetting(settingName) {
+    /* istanbul ignore if: difficult to test settings prior to bootstrap */
```

Comment:
we _could_ test this, but this would occur if we could force a logout prior to the initial bootstrap..  it's possible, but it's not easy and this particular scenario would break our coverage reports..  so we'd have to manufacture some other org setting to test it.. not impossible, but seems like overkill.